<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProbabilityCalcForm</class>
 <widget class="QDialog" name="ProbabilityCalcForm">
  <property name="windowTitle">
   <string>Расчет начальной вероятности</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelLkp">
     <property name="text"><string>Координаты LKP:</string></property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lkpCoordinates"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelErrorRadius">
     <property name="text"><string>Погрешность (NM):</string></property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="errorRadius">
     <property name="minimum"><double>0.1</double></property>
     <property name="maximum"><double>100.0</double></property>
     <property name="value"><double>5.0</double></property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelWeather">
     <property name="text"><string>Учет метеоусловий:</string></property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="weatherCheckBox"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="buttonGenerate">
     <property name="text"><string>Сгенерировать карту</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>